manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/minio.yaml

```diff
@@ -38,13 +38,17 @@ spec:
         - args:
             - gateway
             - gcs
+            - {{ .Values.managedstorage.gcsProjectId }}
```

Comment:
it's temp for unlock work and will be changed to fetch it via GCE metadata server.